danuluma/dannstore
# #161764842 add multiple items in a sales record

#### What does this PR do?
add the ability to create sale records with multiple items

#### Description of Task to be completed?
sales should have multiple items

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Run ```python3 -m unittest``` to run tests
* They should all pass
* Use postman to add several books
* Create a sale record passing in multiple books as a list


#### What are the relevant pivotal tracker stories?
[#161764842](https://www.pivotaltracker.com/story/show/161764842)
